FAQ.md

```diff
@@ -58,18 +59,18 @@ I plan on adding all skills, minigames and bosses from OSRS to the bot. Anything
 ##### Where can I make suggestions for the bot?
 Join http://support.oldschool.gg/ and post your suggestion in the suggestions channel and we'll consider adding it.
 
-##### Can you stack clues on Old School Bot?
+##### Can you stack clues on the Old School Bot?
```

Comment:
I think this change isn't correct, it's not "the" old school bot, it's just old school bot

Reply:
Ah, sorry. Just saw that on the Minions page too. Remove the addition of `the` on line 61 and pushed changes to the branch. Thanks.